--- a/src/plugins/accessible/quick/quick.pro~	2012-03-15 07:44:50.000000000 +0100
+++ b/src/plugins/accessible/quick/quick.pro	2012-03-15 09:23:53.000000000 +0100
@@ -5,7 +5,7 @@ load(qt_plugin)
 include ($$PWD/../shared/qaccessiblebase.pri)
 
 QT += core-private gui-private v8-private qml-private quick-private
-DESTDIR = $$QT.gui.plugins/accessible
+DESTDIR = $(INSTALL_ROOT)/$$QT.gui.plugins/accessible
 
 QTDIR_build:REQUIRES += "contains(QT_CONFIG, accessibility)"
 
